ringbuf/reader.go

```diff
@@ -22,12 +21,9 @@ var (
 	errBusy    = errors.New("sample not committed yet")
 )
 
-var ringbufHeaderSize = binary.Size(ringbufHeader{})
-
 // ringbufHeader from 'struct bpf_ringbuf_hdr' in kernel/bpf/ringbuf.c
 type ringbufHeader struct {
-	Len   uint32
-	PgOff uint32
```

Comment:
Nit: do `_ uint32` instead to make it clear that the struct in the kernel is larger? Otherwise the comment is incorrect. Also doesn't match `BPF_RINGBUF_HDR_SZ`.

Reply:
Done